### DISCUSSION TITLE
Filter collection based on term

### DISCUSSION THREAD

**Discussion by pushpanda** [statamic/cms#5676] [Troubleshooting] [answered]
I'have to filter a collection based on a taxonomy "category". i use the term fieldtype to let the editor pick one term of the taxonomy. it gets saved correctly

`category: brust`

but i'm unable to even dump the value in my antlers view. i always get NULL: 

`{{ category | dump }} or {{ category:slug | dump }}`

what am i missing? the strange thing is, everything worked fine in the past. the only thing that changed (didn't touch the content nor the templates for a month) was, an upgrade to Statamic 3.3.x

**Comment by edalzell** [comment]
is the taxonomy hooked up to the collection?

**Reply by pushpanda** [reply]
Yes, for sure. the taxonomy is part of the collection yaml

```yaml
taxonomies:
  - category
```

i have upgraded from 3.2 to 3.3.x, using a multisite setup

**Comment by pushpanda** [comment, marked as answer]
i think i found "the" solution or a workaround now:

the handle of my term fieldtype is the same as the handle of the taxonomy. so i think that is the problem.

as soon as i change the handle of the field, i can access the field in my views correctly
